Header header
float64 latitude
float64 longitude
float32 altitude
float64 utm_easting
float64 utm_northing
int32 zone 
string letter